QModManager/PirateCheck.cs

```diff
@@ -34,46 +34,59 @@ private void Start()
 
                 GetVideo();
                 //DontDestroyOnLoad(this);
-            }*/
+            }
 
             private void Update()
             {
                 RuntimeManager.GetBus("bus:/master").setMute(true);
                 UWE.Utils.alwaysLockCursor = true;
                 UWE.Utils.lockCursor = true;
-                SceneManager.GetActiveScene().GetRootGameObjects().Do(go =>
+                string[] bannedGOs =
```

Comment:
For how this is being used, make this `static readonly HashSet<string> BannedGameObjectNames` instead.

Reply:
Addressed in commit 58e5345